src/proofPoints.ts

```diff
@@ -27,6 +27,42 @@ interface ProofPointIssueResult {
     proofPointObject: ProofPoint;
 };
 
+enum ProofPointValidity {
+    /**
+     * The proof point object is badly formed. The proof point is invalid.
+     */
+    InvalidBadlyFormed,
```

Comment:
Are all the `Invalid` prefixes needed? Could they just be grouped and documented? Make `Valid` = 0 if possible?

Reply:
the danger with making valid 0, is 0 could be a default value in some contexts and id rather fail safely in that case

Reply:
I added the Invalid prefix to make it absolutely clear that the status implies that the thing is invalid. There shouldn't be any room for (mis)interpretation